src/pages/somi/Feeds/Feeds.js

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Comment from '../Comment/Comment';
+import '../Feeds/Feeds.scss';
+
+class Feeds extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      commentValue: '',
+      commentList: [],
+      likedFeed: false,
+      isLiked: false,
+    };
+  }
+
+  componentDidMount() {
+    this.setState({
+      commentList: this.props.feedList.comment,
+    });
+  }
+
+  handleCommentInput = e => {
+    this.setState({
+      commentValue: e.target.value,
+    });
+  };
+
+  submitComment = e => {
+    const { commentList, commentValue } = this.state;
+    e.preventDefault();
+    this.setState({
+      commentList: [
+        ...commentList,
+        {
+          commentId: commentList.length + 1,
+          commentUserName: 'somangoi',
+          content: commentValue,
+          isLiked: false,
+        },
+      ],
+      commentValue: '',
+    });
+  };
+
+  deleteComment = e => {
+    let item = e.target;
+
+    this.setState({
+      commentList: this.state.commentList.filter(el => {
+        return el.commentId !== Number(item.id);
+      }),
+    });
+  };
+
+  likeComment = e => {
+    const item = e.target;
+    if (this.state.isLiked === false) {
+      this.setState({
+        isLiked: true,
+      });
+      item.className = 'likeBtn fas fa-heart';
+    } else {
+      this.setState({
+        isLiked: false,
+      });
+      item.className = 'likeBtn far fa-heart';
+    }
+  };
```

Comment:
```suggestion
  likeComment = e => {
    const { isLiked } = this.state
    
    this.setState({ isLiked: !isLiked });
  };
```
likeComment 쪽에서는 isLiked의 상태를 토글만 해주시고
```js
<div className={`likeBtn far fa-heart ${isLiked ? "far" : "fas" }`} />
```
변화하는 클래스명만 dom 쪽에서 조작해주면 되지 않을까 싶어요!